css/_global.css

```diff
@@ -44,4 +44,22 @@ a {
 
 img {
     max-width: 100%;
+}
+
+button {
+  padding: 0;
+  border: none;
+  font: inherit;
+  color: inherit;
+  background-color: transparent;
+  cursor: pointer;
```

Comment:
I recommend you to add default hover state for button. For example:
button::hover {
 opacity: 0.8;
}